src/telemetry_util.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <format>
+#include <string_view>
+
+#include "string_util.h"
+
+extern "C" {
+#include <ngx_http.h>
+}
+
+namespace datadog::nginx {
+
+constexpr std::string_view relative_filepath(
+    std::string_view absolute_filepath) {
+  constexpr std::string_view prefix = "nginx-datadog";
+  size_t pos = absolute_filepath.find(prefix);
+  return pos != std::string_view::npos ? absolute_filepath.substr(pos)
+                                       : absolute_filepath;
+}
+
+inline std::string make_current_frame(ngx_http_request_t *request,
+                                      std::string_view file, size_t line,
+                                      std::string_view function) {
+  // clang-format off
+  return std::format("Exception catched:\n"
```

Comment:
```suggestion
  return std::format("Exception caught:\n"
```

Reply:
That's an interesting one. `catched` is actually valid English and commonly used before 19th century, which is when `caught` started to replaced it.

Reply:
Oh, so every time I said catched I was not making any mistake. Nice to know haha